docs/root/start/start.rst

```diff
@@ -12,6 +12,18 @@ These examples use the :ref:`v3 Envoy API <envoy_api_reference>`, but use only t
 feature of the API, which is most useful for simple requirements. For more complex requirements
 :ref:`Dynamic Configuration <arch_overview_dynamic_config>` is supported.
 
+.. _install:
+
+Building and installation
+-------------------------
+
+.. toctree::
+    :maxdepth: 2
+
+    building
+    install/ref_configs
```

Comment:
IMO the ref_configs are somewhat unrelated to building and going through the initial examples. Can you move them somewhere else?

Also, I think start.rst should be a raw TOC page without any text, as it's going to be the entry point to a lot of content. WDYT about moving all of the text in here into smaller independent pages that we can more easily sequence for the right flow?

Reply:
> ref_configs are somewhat unrelated to building

yep, most of the stuff in the `install` folder probs needs to be moved

> I think start.rst should be a raw TOC page without any text

i think probably yes, but atm it is giving the section structure

could we make these changes after landing this?

the redirects, can be chained, so its ok to move and move again, and im  quite up for changing this incrementally

overall tho, defo we want to be able to create a flow through the docs, and the ability to hone in on specific solutions (and SEO to get there etc), so im for moving this around more

Reply:
ill look and see what i can do with `ref_configs` in the meantime

Reply:
Yeah sure I'm fine to do this incrementally as long as we agree on the overall direction. LMK if you want to play with this more or merge.

Reply:
lets merge it for now, and ill make some changes when i add the install page